assets/javascripts/admin_user.js

```diff
@@ -37,4 +37,29 @@ function setup_admin_user() {
             }
         });
     });
+
+    window.deleteUser = function(id){
+        if (confirm("Are you sure you want to procede deleting this user?")) {
+            $.ajax({
+                url: '/api/v1/user/' + id,
+                method: 'DELETE',
+                dataType: 'json',
+                success: function() {
+                  addFlash('info', 'The user was deleted successfully.');
+                  var row = window.admin_user_table.row($("#user_" + id));
+                  row.remove().draw();
+                },
+                error: function(xhr, ajaxOptions, thrownError) {
+                  var error_message = xhr.responseJSON.error;
+                  switch (xhr.status) {
+                      case 401:
```

Comment:
This shouldn't be necessary as only admins can access this route

Reply:
Yes - and even if not it makes more sense to hide the delete button for non-admins.

Reply:
Done